agir/front/components/dashboardComponents/Footer.js

```diff
@@ -225,11 +226,25 @@ const FooterWrapper = styled.footer`
 `;
 
 export const Footer = (props) => {
-  const { hideBanner, isSignedIn, is2022, routes, desktopOnly } = props;
+  const {
+    hideBanner,
+    isSignedIn,
+    is2022,
+    routes,
+    desktopOnly,
+    displayOnMobileApp,
+  } = props;
+
+  const { isMobileApp } = useMobileApp();
   const hasBanner = !hideBanner && isSignedIn === false;
 
+  if (useMobileApp().isMobileApp && !displayOnMobileApp) return null;
```

Comment:
Pas besoin de rappeler le hook, vu qu'on a dejà le résultat ligne 238 :
```suggestion
  if (isMobileApp && !displayOnMobileApp) return null;
```